libs/langchain/langchain/llms/titan_takeoff.py

```diff
@@ -10,8 +10,10 @@
 
 
 class TitanTakeoff(LLM):
-    port: int = 8000
```

Comment:
note this is a breaking change. if you wanted you could continue to support both port and base url for backwards compatibility, something like
```python
port: Optional[int] = None
base_url: str = "http://localhost:8000"

@root_validator(pre=True)
def check_url(cls, values: Dict) -> Dict:
    if "port" in values and "base_url" in values:
        raise ValueError("only specify one...")
    elif "port in values":
        warning.warn("port is deprecated, use...")
        port = values["port"]
        values["base_url"] = f"http://localhost:{port}"

     return values
```


on the other hand this is a pretty new integration so maybe better to just make the breaking change now. will let you decide